config/crd/bases/core.oam.dev_applicationscopes.yaml

```diff
@@ -3,18 +3,14 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  annotations:
-    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
Most of the changes in this PR is related to controller-gen, make sure you're using 0.2.4.

Reply:
fix the issue by bumping to 0.2.5